### ISSUE TITLE
[Question]: discoverServices Time out after 15s

### ISSUE THREAD

**Issue by mpillawa** [boskokg/flutter_blue_plus#562]
### FlutterBluePlus Version

1.15.5

### Flutter Version

3.13.0

### What OS?

Android

### OS Version

13

### Ask your question

After connecting a device I try to discover the services, but I do get a Time out. 
The connect seems to work properly, the log shows "connected".

Any hints?

`        
await selectedDevice!.connect();
List<BluetoothService> services = await selectedDevice!.discoverServices();
`

Thanks.

### Logs

```shell
FlutterBluePlusException: discoverServices: (code: 1) Timed out after 15s
```

**Comment by shuizhuyu1015** [comment]
same issue！
It appears after turning off the Bluetooth adapter, and then turning on the adapter to rescan.
The log shows “already connected”，but no device is actually connected.

**Comment by mpillawa** [comment]
BTW: same code works on iOS

**Comment by chipweinberger** [comment]
post full logs

**Comment by chipweinberger** [comment]
should be fixed in 1.15.7

open a new issue with full logs if not.
